tests/framework/lookup/test_lookup.py

```diff
@@ -151,9 +158,10 @@ def test_interpolated_tables__only_categorical_parameters(
 
     base_config.update({"population": {"population_size": 10000}})
 
-    simulation = InteractiveContext(components=[TestPopulation()], configuration=base_config)
+    component = TestPopulation()
```

Comment:
Should probably try using BasePopulation like I was doing in pop refactor.

Reply:
It doesn't work naively. Also, I believe those cases were in VPH, whereas this is vivarium.